src/composables/events/DialogEvent.js

```diff
@@ -0,0 +1,35 @@
+import { Subject } from 'rxjs';
+
+/**
+ * Represent a Event object to emit and to receive to close or open a dialog.
+ * @typedef {Subject} DialogEvent
+ */
+const DialogEvent = new Subject();
```

Comment:
As discussed, simply exporting the subject without additional functions would limit the risk of confusion between rxjs and vanilla events further down the road

Reply:
done :)